src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve2.cs

```diff
@@ -767,7 +740,7 @@ internal Arm64() { }
         ///   USQADD Ztied1.H, Pg/M, Ztied1.H, Zop2.H
         ///   USQADD Ztied1.H, Pg/M, Ztied1.H, Zop2.H
```

Comment:
Nit: We don't need two identical USQADDs in the comment block here. It'd be nice to clean that up (and any other instances in the file, plus the notsupported file too)